### ISSUE TITLE
All changes made to save are reverting when injected.

### ISSUE THREAD

**Issue by doogieboogie** [kwsch/NHSE#592]
I'm having major issues with NHSE since using the latest build. The biggest one being the fact that any changes I make to my save aren't applied despite pressing save. I get the usual message that the save was successful, but when I inject the save nothing has changed. And these are minor but there is this green boarder around the UI now and I can no longer drag and drop my save folder to open it any more. Let me know if you need more details, it could just be me since I'm not seeing anyone else with these problems but at the moment NHSE is unusable to me and I'd like some help.

**Comment by berichan** [comment]
Are you transferring and restoring correctly? These type of issues are usually beyond the scope of the program, NHSE is a save editor and it works for this, saving/restoring correctly is for the end-user to take care of.

> there is this green boarder around the UI now and I can no longer drag and drop my save folder to open it any more

I can't repro this either with the latest build, can you provide a screenshot/video?

**Comment by doogieboogie** [comment]
I'll take screenshots in a moment but I just wanted to post an update. I discovered something while moving files. If I edit the main.dat in any way via NHSE, then save, and then go back to the file the save is in and either move the main.dat and its accompanying files to another folder or change the name of the folder the changes are reverted. When I noticed this, I went into my back ups folder and tried an older save. I edited it and changed the name of the folder and opened the file. This time I got an actual error code "The file '*directory*\landname.dat' already exists." Obviously directory refers to where landname.dat is located in my drive I just didn't want to write it out, but you get the idea. I think that's why when I inject the save the changes are reverting because I have to move the file over to my switch to inject it. It sound really weird I know, I wish I knew what was happening.

**Comment by kwsch** [comment]
The program does not track where you move a file after you load it.

The save operation only writes a new file back to the same file path was originally loaded from.

**Comment by doogieboogie** [comment]
I don't know what's going on then. The program was working fine until I installed the latest update. The other things I noticed wrong is that I can't drag and drop files anymore and this green boarder. I don't know if either are associated with my issue, but all this began happening at the same time. Here is what that green boarder looks like for reference, if this is intended then never mind.

I don't know if it's worth mentioning but I'm also on Windows 7
![boarder](https://user-images.githubusercontent.com/76706560/154198672-31dd7ef1-c970-4061-83c8-2e9c0fb4d917.png)
.
